SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTest.h

```diff
@@ -28,8 +28,11 @@ class SPATIALGDKFUNCTIONALTESTS_API AEventTracingTest : public ASpatialFunctiona
 	virtual void PrepareTest() override;
 
 protected:
+	FName ReceivePropertyUpdateEventName = "unreal_gdk.receive_property_update";
 	FName ReceiveOpEventName = "worker.receive_op";
 	FName MergeComponentUpdateEventName = "unreal_gdk.merge_component_update";
+	FName ProcessRPCEventName = "unreal_gdk.process_rpc";
+	FName SendRPCEventName = "unreal_gdk.send_rpc";
```

Comment:
Would it make sense to move the sending names here as well?
Or more generally in the event tracing code, would it make sense to expose these names somewhere as FName constants?

Reply:
Yeah we could do that. I think the only use will be in testing but we can add them as constants if you are ok with that.